### DISCUSSION TITLE
CORS error preventing image attachment to PR

### DISCUSSION THREAD

**Discussion by tobynorth** [community/community#45164] [Projects and Issues]
### Select Topic Area

Bug

### Body

For the past couple of months, whenever I've tried to attach an image to a new or existing PR or PR comment, I get the error message "Something went really wrong, and we can’t process that file." In the console, I see the following error (in Firefox, although I encountered a similar error in Chrome):

`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://github-production-user-asset-6210df.s3.amazonaws.com/`

This _only_ seems to affects image files -- I seem to be able to attach text files and PDFs just fine, for example.

Any ideas about whether this is a Github bug or an issue on my end somehow, and what can be done about it either way? For the record, I have no browser extensions/plugins/etc. installed whatsoever.

**Comment by jge162** [comment]
@tobynorth It's possible that this is an issue on GitHub's end, as they may not have properly configured the CORS (Cross-Origin Resource Sharing) headers on their S3 bucket. I can’t say for sure tbh. 

 it's also possible that there is an issue with your browser settings or network configuration that is causing the problem.

You can try clearing your browser's cache and cookies, and then reload the page

**Reply by tobynorth** [reply]
Thanks for the suggestion @jge162. Clearing cookies and site data didn't work, _but_ it did work when I switched from using my home WiFi network to using my phone's satellite connection.

So I guess it's a network problem! Any idea why my WiFi network might be giving me this particular issue and how to fix? 😕 No one has intentionally made any config changes to it in recent memory.

**Reply by jge162** [reply]
Glad to hear. 👍
